CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Cromwell Change Log
 
+## 27
```

Comment:
@Horneth Double checking that this is going in after the release (and thus this number is correct)